aliasing/api/combat.py

```diff
@@ -151,11 +151,10 @@ def __init__(self, combatant: Combatant, hidestats=True):
         self._update_effects()
         # Type-specific Properties
         self._race = None
-        self._creature_type = None
         self._monster_name = None
+        self._creature_type = combatant._creature_type
```

Comment:
Since SimpleCombatant is a subclass of AliasStatBlock, which is now providing the `creature_type` property, this and the accessor on L203 can be deleted